SQL/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace SQL
+{
+ 
+1: create table toys (id int, name varchar(100), price float, quantity int);
+2: insert into toys values (1, 'dragon', 20, 20.00);
+3: insert into toys values (2, 'bob the builder', 15, 15.00);
+4: insert into toys values (3, 'pokemon',25, 25.00);
+5: insert into toys values (4, 'X-man', 5, 5.00);
+6: insert into toys values (5, 'saliorman' 18, 18.00);
+7: insert into toys values (6, 'GI Joe', 20, 20.00);
+8: insert into toys values (7, 'Barbie', 30, 19.99);
+9: insert into toys values (8, 'thomas the train', 60, 220.00);
+10: insert into toys values (9, 'tom and jerry', 80, 120.00);
+11: insert into toys values (10, 'batman', 100, 70.00);
+
+12: alter table toys add column department_id int
+
```

Comment:
I was hoping to see an update statement where you would set the department for the items you already added.